src/tasks/payload/payload_task.c

```diff
@@ -30,6 +30,65 @@ void payload_task_init(slate_t *slate)
     // NOTE: Turning on payload is handled by command_parser
 }
 
+bool ping_command(slate_t *slate)
+{
+    char packet[] = "[\"ping\", [], {}]";
+    int len = sizeof(packet) - 1;
+    payload_uart_write_packet(slate, packet, len, 999);
```

Comment:
We can construct a new function `bool send_payload_exec(string)` function that handles:

sending a well-formatted string of payload into the payload through UART. This bit of logic is duplicated in the `try_execute_payload_command` innermost if statement.

Note that some changes will also be required for the try_execute_payload_command to keep its previous behavior consistent (i.e. when send_payload_exec returns true we [remove the command from the queue](https://github.com/stanford-ssi/samwise-flight-software/blob/e0e8dc18b07ecae86202d7e5930694bb0f33a67a/src/tasks/payload/payload_task.c#L56)).

This will replace this function here and wrap it with error checking. If the sending wasn't successful, we should just return false at this point without waiting for the receiving.